components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -52,6 +53,12 @@ const (
 	BEBPublishEndpointForPublisher  = "/sap/ems/v1/events"
 
 	reconcilerName = "backend-reconciler"
+
+	certificateSecretNamespace = "kyma-system"
+	certificateSecretName      = "eventing-webhook-server-cert"
+	mutatingWHName             = "subscription-mutating-webhook-configuration"
+	validatingWHName           = "subscription-validating-webhook-configuration"
```

Comment:
(nit) maybe make take this from configuration defined in the helm charts as it is used in the mutating and validating webhook configurations.

Reply:
done